apps/emqx/src/emqx_schema.erl

```diff
@@ -2910,7 +2918,12 @@ parse_server(Str, Opts) ->
 %% @doc Parse comma separated `host[:port][,host[:port]]' endpoints
 %% into a list of `{Host, Port}' tuples or just `Host' string.
 -spec parse_servers(undefined | string() | binary(), server_parse_option()) ->
-    [{string(), port_number()}].
+    [
```

Comment:
IMO, this change confused the result, let me must be careful when I am going to use it

Reply:
That's a fair point.  I'll try to refactor it into a separate function to deal with the schemes.

Reply:
Or maybe improving the return type would be better: returning a map with semantic keys rather than a tuple would be better, I believe.